src/Dashboard/Resources/Filter/Filter.tsx

```diff
@@ -1,99 +1,114 @@
-import React, { useState, useEffect } from 'react';
+import { useState, useEffect } from 'react';
 
 interface Props {
-  onUpdate: (tags: string[]) => void;
+  onUpdate: (tags: string[], count: {'unit': number, 'type': number}) => void;
 }
 
-const Filter = ({ onUpdate }: Props) => {
+let prevTags = [''];
 
+const Filter = ({ onUpdate }: Props) => {
   const [selectedTags, setSelectedTags] = useState(['']);
+  const [selectedTagsCount, setSelectedTagsCount] = useState({'unit': 0, 'type': 0})
 
-  const handleTagSelect = (tag: string) => {
+  const handleTagSelect = (tag: string, tagClass: string|null) => {
     const isTagSelected = selectedTags.includes(tag);
 
     if (isTagSelected) {
       setSelectedTags(selectedTags.filter((t) => t !== tag));
+      if (tagClass === 'unit' || tagClass === 'type') {
+        let newTagCount = selectedTagsCount;
+        newTagCount[tagClass] -= 1;
```

Comment:
This is a cool way of keeping track of this!

Reply:
:D